lib/router.js

```diff
@@ -637,22 +637,21 @@ class Router {
 Router.prototype._createNewApiRoot = function (node, spec, scope) {
     const specRoot = Object.assign({}, spec);
     // Make sure the spec has the standard properties set up.
-    specRoot.swagger = spec.swagger || '2.0';
-    specRoot.definitions = spec.definitions || {};
-    specRoot.securityDefinitions = spec.securityDefinitions || {};
+    specRoot.openapi = spec.openapi || '3.0.1';
+    specRoot.components = spec.components || {};
     specRoot['x-default-params'] = spec['x-default-params'] || {};
     specRoot.tags = spec.tags || [];
 
     delete specRoot['x-route-filters'];
 
     // Reset paths. These are going to be built up during path setup.
     specRoot.paths = {};
-    specRoot.basePath = scope.prefixPath;
+    specRoot.servers = [{ url: scope.prefixPath }];
 
     node.setChild({ type: 'meta', name: 'apiRoot' }, new Node({
         specRoot,
         methods: {},
-        path: `${specRoot.basePath}/`,
+        path: `${specRoot.servers.url}/`,
```

Comment:
`servers` is an array, so you need `servers[0].url` here

Reply:
Done.